src/main/bash/sdkman-config.sh

```diff
@@ -26,5 +26,5 @@ function __sdk_config() {
 		return 1
 	fi
 
-	"${editor[@]}" "${SDKMAN_DIR}/etc/config"
+	"${editor[@]}" "${SDKMAN_DIR}/.sdkmanrc"
```

Comment:
This is pointing to `${SDKMAN_DIR}/.sdkmanrc`, not `${HOME}/.sdkmanrc`

Reply:
Ops, my bad!

Reply:
We need to think carefully about which file has precedence when editing. I don't think `$HOME/.sdkmanrc` should take precedence.

Reply:
Changed back to etc/config then.